From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Fri, 2 Dec 2022 01:18:14 +0100
Subject: Enable PermuteTLSExtensions by default

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html

# ------------------------ >8 ------------------------
# Do not modify or remove the line above.
# Everything below it will be ignored.
#
# Conflicts:
#	net/base/features.cc
---
 net/base/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -130,8 +130,8 @@ BASE_FEATURE(kTLS13KeyUpdate,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPermuteTLSExtensions,
-             "PermuteTLSExtensions",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             "PermuteTLSExtensions",               // enabled by default
+             base::FEATURE_ENABLED_BY_DEFAULT);    // in Bromite
 
 BASE_FEATURE(kPostQuantumKyber,
              "PostQuantumKyber",
-- 
2.39.2

